spec/migrations/20190617191109_move_awx_credentials_to_authentications_spec.rb

```diff
@@ -97,7 +97,8 @@
         :type => "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::VaultCredential"
       )
       expect(auth.manager_ref).to be_nil
-      expect(described_class).to_not receive(:pg_connection)
+      allow(PG::Connection).to receive(:new).and_call_original
+      expect(PG::Connection).to_not receive(:new).with(a_hash_including(:dbname => "awx"))
```

Comment:
Again, avoid expectations on the migration class constant as it's been removed and re-loaded by the time these expectations are reached.